index.html

```diff
@@ -35,8 +34,19 @@
         <div style="background-color: #f8f6e7;">
             <img class="home-section-pic" src="assets/home-section2.jpeg">
             <div class="intro">
-                <h1>Our community believes in learning by teaching</h1>
-                <p>Some context from some reliable source about learning by teaching</p>
+                <h1 class="home-section-heading">EDZU</h1>
+                <h3 class="tag-line">Learning by teaching</h3>
```

Comment:
This should be able to be h2 I believe. Head tags should go in order - h1, h2, h3, h4...
```suggestion
                <h2 class="tag-line">Learning by teaching</h2>
```

Reply:
Done!